src/corona_hakab_model/generation/circles_consts.py

```diff
@@ -93,12 +93,34 @@ def get_age_distribution(self):
 
     def get_connection_types_prob_by_age(self):
         return {age: self.connection_type_prob_by_age_index[i] for i, age in enumerate(self.ages)}
-
+    
     # overriding hash and eq to allow caching while using un-hashable attributes
     __hash__ = object.__hash__
     __eq__ = object.__eq__
 
 
+    def get_required_adult_distributions(self):
```

Comment:
Please add documnetation for this function. It's same is also not indictive

Reply:
added